pkg/services/resource.go

```diff
@@ -127,6 +137,38 @@ func (s *sqlResourceService) Replace(ctx context.Context, resource *api.Resource
 	if eErr != nil {
 		return nil, handleUpdateError("Resource", err)
 	}
+
+	return updated, nil
+}
+
+func (s *sqlResourceService) UpdateStatus(ctx context.Context, resource *api.Resource) (*api.Resource, *errors.ServiceError) {
+	if !DisableAdvisoryLock {
```

Comment:
when would locks be disabled?

better to inject a _mock_ of this service through DI than to set a flag by env.

Reply:
Looks like the flag will be removed in https://github.com/openshift-online/maestro/pull/7
I will remove this, too.